src/Button/Button.stories.js

```diff
@@ -42,14 +51,24 @@ storiesOf('Button', module)
       ['undefined', ...Object.keys(icons)],
       'calendar'
     );
+    const width = number('Width', NaN);
```

Comment:
I think it's better to set default value instead of `Nan`. Check if you can change `Nan` value in mobile playground.

Reply:
My thinking was that you can see it defaults to being full-width when you do not specify any width. But I guess it's weird to see `NaN` as a default value 👍